src/webnn_native/dml/GraphDML.cpp

```diff
@@ -912,6 +912,14 @@ namespace webnn_native { namespace dml {
             }
             output =
                 ::dml::AveragePooling(input, strides, windowSizes, startPadding, endPadding, false);
+        } else if (pool2d->GetType() == op::Pool2dType::kL2Pool2d) {
+            uint32_t length = SizeOfShape(inputDims);
+            std::vector<float> constant(length, 2);
+            auto pow = ::dml::Pow(input, BindingConstant(DML_TENSOR_DATA_TYPE_FLOAT32, inputDims,
+                                                         constant.data(), sizeof(float) * length));
+            auto avgPool2d =
+                ::dml::AveragePooling(pow, strides, windowSizes, startPadding, endPadding, false);
```

Comment:
Maybe there is performance issue for DML_OPERATOR_LP_POOLING, should we start to use DirectML directly instead of the wrapper DirectMLX?

Reply:
Ok, I will start to investigate. Thanks! @fujunwei

Reply:
Added a new [commit](https://github.com/webmachinelearning/webnn-native/pull/171/commits/b5636c0bdac479cef3d5de177e12527c104d6e05) for annotating emulated l2pool2d. PTAL, thanks! @fujunwei